### ISSUE TITLE
Caliburn.Micro issue

### ISSUE THREAD

**Issue by Jankiasati** [Caliburn-Micro/Caliburn.Micro#905]
Hi Team,

I am working on dot net project where i migrated my dot net framework 4.6.1 to 4.8 along with visual studio 2015 to visual studio 2022.

when am doing so am getting warning for few project component. I attached the warning SS.

I tried to upgrade Nuget package(Caliburn.Micro and Caliburn.Micro.Plateform) , Caliburn.Micro.Plateform is getting upgraded to latest version but Caliburn.Micro is not getting upgraded to latest version.

I have caliburn.Micro library which is targeting 4.5 dotnet version when i try to change the target from 4.5 to 4.8.

4.8 dotnet fraemwork is not getting listed in dropdown.

could you help me.
![Target Framework](https://github.com/user-attachments/assets/07a5106d-8764-465d-ac58-1d88d166221f)

**Comment by Jankiasati** [comment]
<img width="572" alt="Screenshot 2024-10-18 104626" src="https://github.com/user-attachments/assets/e2ce723b-8a2d-4f09-b0a4-7968510d8cd8">

**Comment by vb2ae** [comment]
Are you using Caliburn.Micro.Core as the nuget package?
https://www.nuget.org/packages/Caliburn.Micro.Core

**Comment by Jankiasati** [comment]
No, we would like to know which version of caliburn.Micro is compatible with .net 4.8 framework.

**Comment by vb2ae** [comment]
the current version of Caliburn.Micro is compatible with .net framework 4.8.   What I recommend is uninstalling caliburn.micro and Caliburn.Micro.Platform and install Caliburn.Micro.Core.    Installing Caliburn.Micro.Core will also install the proper version of Caliburn.Micro.   I suspect the error is version mismatch with Caliburn.Micro.Platform

**Comment by vb2ae** [comment]
Actually I am seeing the same on the Features.Wpf sample app.    Do you have this warning about System.Net.Http also?

![image](https://github.com/user-attachments/assets/3c8d4761-9a76-4b0a-a5e2-3fad99340ff4)

**Comment by Jankiasati** [comment]
No, I did not get the error which you have mentioned.

When we are upgrading to latest version of Caliburn.Micro(4.0.212) it is adding Caliburn.Micro.Core ,Caliburn.Micro.PlateForm.Core Package references. 

And it's keeping our existing Package references Caliburn.micro to the older version(2.0.0.5) and only upgrading Caliburn.Micro.Plateform to the latest version 4.0.0.0

After that if am building my project am getting following errors.
the type "sample " exist in both caliburn.micro , verson = "2.0.0.5" and caliburn.micro.core , version =="4.0.0.0".

if am removing caliburn.micro and keeping caliburn.micro.core refrences to my project am getting the error which i added in "Error SS1"
<img width="505" alt="Error SS1" src="https://github.com/user-attachments/assets/4a83ef15-7947-46a1-9a21-b724462b10e1">

And if am keeping caliburn.micro and removing caliburn.micro.core to our project we are getting the error which i added in ErrorSS2

<img width="466" alt="Error SS2" src="https://github.com/user-attachments/assets/12dad581-6496-4379-ba2f-813d75cb9c59">

**am trying to migrate dotnet framework version from 4.6 to 4.8.1 and we are not migrating to .net core version.
so please tell us the  caliburn.micro package which is compatible with  .net 4.8.1 framework.****

if we are going with the latest versions it is breaking our existing changes as function signature is changed to async

**Comment by vb2ae** [comment]
Started with a new wpf app and added Caliburn.Micro 3.  I then upgraded Caliburn.Micro to the latest.  This is what I see in packages.config

<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="Caliburn.Micro" version="4.0.212" targetFramework="net48" />
  <package id="Caliburn.Micro.Core" version="4.0.212" targetFramework="net48" />
  <package id="Microsoft.Xaml.Behaviors.Wpf" version="1.1.19" targetFramework="net48" />
  <package id="System.Runtime.Serialization.Primitives" version="4.3.0" targetFramework="net48" />
</packages>

What is in your packages.config?  Wondering if maybe there is something that requires the old version of Caliburn.Micro which prevents the upgrade to the newer version

**Comment by Jankiasati** [comment]
Hello, My configuration is also same. Could you check is it adding Caliburn.Micro references in your project with latest version because my configuration is getting updated but it's not adding reference to the project.

**Comment by Jankiasati** [comment]
Hi.. Please clarify.

**Comment by vb2ae** [comment]
After you try and update caliburn.micro look in the output window output for package manager.   Is there anything saying it could not update the package with a reason
